kraken/node_actions/openstack_node_scenarios.py

```diff
@@ -0,0 +1,133 @@
+import sys
+import time
+import logging
+import subprocess
+import requests
+import kraken.kubernetes.client as kubecli
+import kraken.invoke.command as runcommand
+import kraken.node_actions.common_node_functions as nodeaction
+from kraken.node_actions.abstract_node_scenarios import abstract_node_scenarios
+
+
+class OPENSTACKCLOUD:
```

Comment:
@paigerube14 I think python [PEP8](https://www.python.org/dev/peps/pep-0008/#class-names) states that class names should be CamelCase.